.buildkite/x-pack/pipeline.xpack.metricbeat.yml

```diff
@@ -154,9 +154,10 @@ steps:
               context: "x-pack/metricbeat: Ubuntu x86_64 fips140=only Unit Tests"
 
       - label: ":ubuntu: x-pack/metricbeat: Go Integration Tests (Module)"
-        key: "mandatory-int-test"
+        key: "x-pack-metricbeat-mandatory-int-test"
         env:
           TEST_TAGS: "oracle"
+          MODULE: ""
```

Comment:
For some reason, `MODULE=kubernetes` and so far I only see that in another pipeline, so `buildkite-agent pipeline upload` seems to do something under the hood and populate variables unexpectedly.